### ISSUE TITLE
unrecognized arguments: --persevere

### ISSUE THREAD

**Issue by PeiQuan1011** [Hank0626/PDF#11]
usage: run_longExp.py [-h] [--random_seed RANDOM_SEED] --is_training
                      IS_TRAINING --model_id MODEL_ID --model MODEL --data
                      DATA [--root_path ROOT_PATH] [--data_path DATA_PATH]
                      [--features FEATURES] [--target TARGET] [--freq FREQ]
                      [--checkpoints CHECKPOINTS] [--seq_len SEQ_LEN]
                      [--label_len LABEL_LEN] [--pred_len PRED_LEN]
                      [--fc_dropout FC_DROPOUT] [--head_dropout HEAD_DROPOUT]
                      [--add] [--wo_conv] [--serial_conv]
                      [--kernel_list KERNEL_LIST [KERNEL_LIST ...]]
                      [--patch_len PATCH_LEN [PATCH_LEN ...]]
                      [--period PERIOD [PERIOD ...]]
                      [--stride STRIDE [STRIDE ...]]
                      [--padding_patch PADDING_PATCH] [--revin REVIN]
                      [--affine AFFINE] [--subtract_last SUBTRACT_LAST]
                      [--decomposition DECOMPOSITION]
                      [--kernel_size KERNEL_SIZE] [--individual INDIVIDUAL]
                      [--embed_type EMBED_TYPE] [--enc_in ENC_IN]
                      [--dec_in DEC_IN] [--c_out C_OUT] [--d_model D_MODEL]
                      [--n_heads N_HEADS] [--e_layers E_LAYERS]
                      [--d_layers D_LAYERS] [--d_ff D_FF]
                      [--moving_avg MOVING_AVG] [--factor FACTOR] [--distil]
                      [--dropout DROPOUT] [--attn_dropout ATTN_DROPOUT]
                      [--embed EMBED] [--activation ACTIVATION]
                      [--output_attention] [--do_predict]
                      [--num_workers NUM_WORKERS] [--itr ITR]
                      [--train_epochs TRAIN_EPOCHS] [--batch_size BATCH_SIZE]
                      [--patience PATIENCE] [--learning_rate LEARNING_RATE]
                      [--des DES] [--loss LOSS] [--lradj LRADJ]
                      [--pct_start PCT_START] [--use_amp] [--use_gpu USE_GPU]
                      [--gpu GPU] [--use_multi_gpu] [--devices DEVICES]
                      [--test_flop] [--log LOG]
run_longExp.py: error: unrecognized arguments: --persevere 96

I got this problem while running the scripts, I am confused and seeking for solution.
